metaseq/logging/meters.py

```diff
@@ -77,22 +77,12 @@ def update(self, val):
         assert type(val) is list
         self.history.extend(val)
 
-    def state_dict(self):
-        return {
-            "history": self.history,
-            "round": self.round,
-        }
-
-    def load_state_dict(self, state_dict):
-        self.history = state_dict["history"]
-        self.round = state_dict.get("round", None)
-
     @property
     def avg(self):
         if type(self.history[0]) is dict:
-            return sum([h["value"] for h in self.history])
+            return sum([h["value"] for h in self.history]) / len(self.history)
         else:
-            return sum(self.history)
+            return sum(self.history) / len(self.history)
```

Comment:
Is this related to removing the logging or is this a bug fix?

Reply:
It's not related to logging. We never use it, but changed it anyway as per the module definition of returning some average.